### PR TITLE
Hotfix releasenotes 

### PR DESCRIPTION
**Pull request by SaikrishnaBairamoni**
This PR is to update release notes for hotfix 4.4.1 and 4.4.2 
# PR Details
## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING**](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/Release_notes.md [hunk 1]
```diff
@@ -1,6 +1,24 @@
...
+- PR 2105: Fixed Light Controlled Intersection (LCI) strategic plugin where the node fails to load parameters from the vehicle calibration directory.
+
+Version 4.4.1, released May 9th, 2023
+----------------------------------------
+
+### **Summary**
+Carma-System release version 4.4.1 is a hotfix release for 4.4.0.
+
+Fixes in this release:
+- Issue 2104: Fixed an exception that can occurs in rare circumstances when the approaching_emergency_vehicle_plugin cannot process all received BSMs from an active ERV at ~10 Hz, where all the Received BSMs that are processed by the approaching_emergency_vehicle_plugin should not result in any exception.
```
Comment (maefromm) [inline docs/Release_notes.md:20]: Update to 

Issue 2104: Fixed an exception that can occurs in rare circumstances when the approaching_emergency_vehicle_plugin cannot process all received BSMs from an active ERV at ~10 Hz. Received BSMs that are processed by the approaching_emergency_vehicle_plugin should not result in any exception.

File: docs/Release_notes.md [hunk 2]
```diff
@@ -1,6 +1,24 @@
 CARMA Platform Release Notes
 ----------------------------
 
+Version 4.4.2, released May 10th, 2023
+----------------------------------------
+
+### **Summary**
+Carma-System release version 4.4.2 is a hotfix release for 4.4.0.
```
Comment (maefromm) [inline docs/Release_notes.md:8]: Update "Carma-System" to "Carma-platform"

File: docs/Release_notes.md [hunk 3]
```diff
@@ -1,6 +1,24 @@
...
+
+Fixes in this release:
+- PR 2105: Fixed Light Controlled Intersection (LCI) strategic plugin where the node fails to load parameters from the vehicle calibration directory.
+
+Version 4.4.1, released May 9th, 2023
+----------------------------------------
+
+### **Summary**
+Carma-System release version 4.4.1 is a hotfix release for 4.4.0.
```
Comment (maefromm) [inline docs/Release_notes.md:17]: Update "Carma-System" to "Carma-platform"

**Review by maefromm — CHANGES_REQUESTED** [review]

File: docs/Release_notes.md (hunk 1 above)
Comment (SaikrishnaBairamoni) [reply docs/Release_notes.md]: updated

**Review by maefromm — APPROVED** [review]

**Pull request merged**